elements/rh-cta/rh-cta.ts

```diff
@@ -37,14 +37,26 @@ function isButton(element: Element): element is HTMLButtonElement {
 }
 
 /**
- * Call to action stands out from regular hypertext links, and is used for linking users to webpages.
+ * A call to action is a styled link that entices users to make a selection.
  *
- * @summary Directs users to other pages or displays extra content
+ * @summary     Directs users to other pages or displays extra content
  *
- * @slot - We expect an anchor tag, `<a>` with an `href`, to be the first child inside `rh-cta` element. Less preferred but allowed for specific use-cases include: `<button>` (note however that the `button` tag is not supported for the default CTA styles).
- *
- * @csspart container - container element for slotted CTA
+ * @slot
+ *              We expect an anchor tag, `<a>` with an `href`, to be the first child inside `rh-cta` element. Less preferred but allowed for specific use-cases include: `<button>` (note however that the `button` tag is not supported for the default CTA styles).
  *
+ * @csspart     container - container element for slotted CTA
+ * @cssprop     {<color>} --rh-cta-color
```

Comment:
- CSS props missing default values 
- Other CSS variables missing
- Table styling/spacing between headers are inconsistent with XD (but I think this will be handled separately since it impacts all element docs)

Otherwise, lgtm!

Reply:
> * CSS props missing default values

Just to elaborate on this:

CSS Props when they are component specific tokens ie: `--rh-cta-color` then we list the default value.

When they are RHDS system tokens such as `--rh-color-surface-lightest` we do not actually need the default values.  Those values are pulled from the tokens project when the table is built.

We will need those system tokens listed as well until we close this issue: https://github.com/RedHat-UX/red-hat-design-system/issues/1030

Reply:
@zeroedin i posted a question above to clarify, for cta there are multiple defaults for --rh-cta-color based on the variants. same for --rh-cta-background-color, --rh-cta-border-color, etc. M just wondering do we need to cover default for each variant for cta or not.

Reply:
@bennypowers i've updated the table for css props. can you review if this and let me know if there any more changes https://deploy-preview-1045--red-hat-design-system.netlify.app/elements/call-to-action/code/
i've used primary variant for default values.

Reply:
looks good